release-common-lib/src/main/java/org/reactome/util/compare/DBObjectComparer.java

```diff
@@ -112,18 +135,24 @@ public static int compareInstances(
 	 * that will count as 1 diff and the elements will NOT be compared.
 	 */
 	public static int compareInstances(
-		GKInstance inst1, GKInstance inst2, StringBuilder sb, int maxRecursionDepth,
+		GKInstance instance1, GKInstance instance2, StringBuilder stringBuilder, int maxRecursionDepth,
 		Predicate<? super SchemaAttribute> customAttributeNameFilter, boolean checkReferrers
 	)
 	{
-		return compareInstances(inst1, inst2, sb, 0, 0, maxRecursionDepth, customAttributeNameFilter, checkReferrers);
+		final int diffCount = 0;
+		final int initialRecursionDepth = 0;
```

Comment:
Similar to above, should this be capitalized and a global variable?

Reply:
Yup, done!